posts/copilot-workspace/index.qmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "Some thoughts after a trial run of GitHub’s Copilot Workspace"
+author:
+  - name: "Joshua W. Lambert"
+    orcid: "0000-0001-5218-3046"
+  - name: "James Mba Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Pratik Gupte"
+    orcid: "0000-0001-5294-7819"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-06-24"
+categories: [AI, GitHub, Copilot Workspace, R, R package, software development]
+bibliography: index.bib
+format:
+  html: 
+    toc: true
+---
+
+
+GitHub recently previewed ‘Copilot Workspace’, which aims to use generative AI to assist software developers. Rather than just giving in-line suggestions, as GitHub copilot does, workspace allows users to map out and execute entire projects. We got early preview access to the tool, so decided to see how well it performed for our needs in Epiverse.
```

Comment:
```suggestion
GitHub recently previewed [‘Copilot Workspace’](https://github.blog/2024-04-29-github-copilot-workspace/), which aims to use generative AI to assist software developers. Rather than just giving in-line suggestions, as GitHub copilot does, workspace allows users to map out and execute entire projects. We got early preview access to the tool, so decided to see how well it performed for our needs in Epiverse.
```

Reply:
Couldn't commit as it was the same line as another commit suggestion. Change made in 259b5de53e460cb09ff1d50999b31907d171e5bd.